packages/web-react/src/components/Tooltip/stories/TooltipDismissibleFloatingUi.tsx

```diff
@@ -0,0 +1,88 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React, { useState } from 'react';
+import {
+  useFloating,
+  autoUpdate,
+  flip,
+  useHover,
+  useFocus,
+  useDismiss,
+  useRole,
+  useInteractions,
+} from '@floating-ui/react';
+import { ComponentStory } from '@storybook/react';
+import Tooltip from '../Tooltip';
+
+const Story: ComponentStory<typeof Tooltip> = () => {
+  const [open, setOpen] = useState(true);
+
+  const closeHandler = () => setOpen(false);
+
+  const { x, y, refs, context, placement } = useFloating({
+    placement: 'top',
+    whileElementsMounted: autoUpdate,
+    middleware: [flip()],
+  });
+
+  const hover = useHover(context, { move: false });
+  const focus = useFocus(context);
+  const dismiss = useDismiss(context);
+  const role = useRole(context, { role: 'tooltip' });
+  const { getReferenceProps, getFloatingProps } = useInteractions([hover, focus, dismiss, role]);
+
+  return (
+    <>
+      <p>
+        The following example is using external library <a href="https://floating-ui.com">Floating UI</a>.
+      </p>
+      <p>🖱 Try scrolling the example to see how Tooltip placement is updated.</p>
+      <div
+        style={{
```

Comment:
What about creating a class name in `.storybook` assets and using it here in the other FloatingUI example?

Reply:
done